.travis.yml

```diff
@@ -1,27 +1,30 @@
 language: php
 
 php:
-  - 5.3
   - 5.4
   - 5.5
   - 5.6
   - hhvm
 
 env:
-  - SYMFONY_VERSION=2.5.*
+  - SYMFONY_VERSION=2.6.* PACKAGE_VERSION=high
 
 matrix:
   include:
+    - php: 5.3.3
+      env: SYMFONY_VERSION=2.3.* PACKAGE_VERSION=low
     - php: 5.5
-      env: SYMFONY_VERSION=2.3.*
+      env: SYMFONY_VERSION=2.3.* PACKAGE_VERSION=high
     - php: 5.5
-      env: SYMFONY_VERSION=2.4.*
+      env: SYMFONY_VERSION=2.4.* PACKAGE_VERSION=high
     - php: 5.5
-      env: SYMFONY_VERSION=2.6.*
+      env: SYMFONY_VERSION=2.5.* PACKAGE_VERSION=high
 
 before_install:
   - composer self-update
```

Comment:
this line should be `composer self-update || true`. As 5.3.3 doesn't have SSL on Travis, Composer can't run self-update.